app/nuxt.config.js

```diff
@@ -72,11 +72,6 @@ export default {
   },
   sentry: {
     dsn: process.env.SENTRY_DSN,
-    publish_release: true,
```

Comment:
Do we need this removal for the fix to work? Any idea what these lines exactly do?

Reply:
This configuration would be used to monitor different releases, but it was wrongly configured (publishRelease instead of publish_release), during my investigation I tried to set this up properly but unfortunately I couldn't make it work.